plugins/services/src/js/containers/pod-instances/__tests__/PodInstancesTable-test.js

```diff
@@ -403,10 +425,25 @@ describe("PodInstancesTable", function() {
             return el.text().trim();
           });
 
+        const options = {
```

Comment:
Shall we extract those options? Maybe creating a wrapper component that would have them as a default?